src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAnnouncementsService.java

```diff
@@ -68,7 +68,7 @@ public UIAnnouncementsService(AnnouncementService announcementService, UserServi
 	 */
 	public void createNewAnnouncement(@RequestBody AnnouncementRequest announcementRequest, Principal principal) {
 		User user = userService.getUserByUsername(principal.getName());
-		Announcement announcement = new Announcement(announcementRequest.getMessage(), user);
+		Announcement announcement = new Announcement("title placeholder", announcementRequest.getMessage(), false, user);
```

Comment:
Shouldn't `"title placeholder"` be replaced with `announcementRequest.getTitle()`?

Reply:
Yeah, but I think it errors out cause we have a not null database constraint on title. I can move this placeholder to the request.